hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/OnlineDsAssignmentStrategy.java

```diff
@@ -130,15 +131,23 @@ public JpaAction createTargetAction(final String initiatedBy, final TargetWithAc
             final List<JpaTarget> targets, final JpaDistributionSet set) {
         final JpaAction result = super.createTargetAction(initiatedBy, targetWithActionType, targets, set);
         if (result != null) {
-            result.setStatus(Status.RUNNING);
+            if (isUserConsentEnabled()) {
+                result.setStatus(Status.WAIT_FOR_CONFIRMATION);
```

Comment:
Should we maybe add another message in case of the confirmation flow. The JpaActionStatus already contains the message: "Assignment initiated by user '{user}'"

Maybe we can add a second message describing:
"Waiting for the confirmation by the device before processing with the deployment"

Reply:
I am not sure if the linked code section is the right place for that. I guess this needs to be defined during action/assignment creation.